controllers/external/util.go

```diff
@@ -87,6 +87,7 @@ type CloneTemplateInput struct {
 }
 
 // CloneTemplate uses the client and the reference to create a new object from the template.
+// Deprecated: use CreateFromTemplate instead. This function will be removed in a future release.
```

Comment:
Let's add the deprecation notice to CloneTemplateInput as well

Reply:
Done.